pkg/registry/images.go

```diff
@@ -109,9 +127,6 @@ func DeleteUnusedImages(appID string) error {
 		if version.KOTSKinds == nil {
 			continue
 		}
-		if version.KOTSKinds.KotsApplication.Spec.AllowRollback {
```

Comment:
rollback support is detected from the latest available version, not the currently deployed one